docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -190,135 +179,139 @@ For this example, you create an instance of the *Data Generator* source connecto
 +
-If you're using the evaluation OpenShift Dedicated environment, click *Register eval namespace* to provision a namespace for hosting the Connectors instances that you create.
+If you're using the evaluation OpenShift Dedicated environment, click *Register eval namespace* to provision a namespace for hosting the {connectors} instances that you create.
 
 . Click *Next*.
 
-. Configure the core configuration for your connector:
-.. Provide a name for the connector.
-.. Type the *Client ID* and *Client Secret* of the service account that you created for connectors and then click *Next*.
-
+. Configure the core configuration for your {connectors} instance:
+.. Type a name for your {connectors} instance.
+.. Type the *Client ID* and *Client Secret* of the service account that you created for {connectors} and then click *Next*.
 . Provide connector-specific configuration. For the *Data Generator*, provide the following information:
 .. *Data shape Format*: Accept the default, `application/octet-stream`.
-.. *Topic Names*: Type the name of the topic that you created for connectors. For example, type *test-topic*.
+.. *Topic Names*: Type the name of the topic that you created for {connectors}. For example, type *test-topic*.
 .. *Content Type*: Accept the default, `text/plain`.
-.. *Message*: Type the content of the message that you want the Connectors instance to send to the Kafka topic. For example, type `Hello World!`.
-.. *Period*: Specify the interval (in milliseconds) at which you want the Connectors instance to send messages to the Kafka topic. For example, specify `10000`, to send a message every 10 seconds.
+.. *Message*: Type the content of the message that you want the {connectors} instance to send to the Kafka topic. For example, type `Hello World!`.
+.. *Period*: Specify the interval (in milliseconds) at which you want the {connectors} instance to send messages to the Kafka topic. For example, specify `10000`, to send a message every 10 seconds.
 
-. Optionally, configure the error handling policy for your Connectors instance.
+. Optionally, configure the error handling policy for your {connectors} instance.
 +
 The options are:
 +
-* *stop* - (the default) The Connectors instance shuts down when it encounters an error.
-* *log* - The Connectors instance sends errors to its log.
-* *dead letter queue* - The Connectors instance sends messages that it cannot handle to a dead letter topic that you define for the Connectors Kafka instance.
+* *stop* - (the default) The {connectors} instance shuts down when it encounters an error.
```

Comment:
In the previous step, each option name and description is separated by a colon. Here, each option name and description is separated by a dash. For consistency, it might be good to use the same separator.

Reply:
I agree